content/context_processors.py

```diff
@@ -1,5 +1,9 @@
-from .models import FooterInfo
+from .models import FooterInfo, MenuNavbar
 
 def footer(request):
 	footer = FooterInfo.objects.all()
 	return {'footer': footer}
+
+def navbar(request):
+	categories = MenuNavbar.objects.filter(parent__isnull=True)
+	return {'categories': categories}
```

Comment:
I havent checked, but are we using any variables anywhere else that are also named `categories`